tickets/src/main/java/no/unit/nva/publication/ticket/update/UpdateTicketHandler.java

```diff
@@ -233,10 +225,19 @@ private List<AssociatedArtifact> updateUnpublishedFiles(Publication publication)
                    .toList();
     }
 
-    private TicketEntry fetchTicketForElevatedUser(SortableIdentifier ticketIdentifier, UserInstance userInstance)
-        throws ForbiddenException {
+    private TicketEntry fetchTicketForElevatedUser(RequestUtils requestUtils)
+        throws ForbiddenException, NotFoundException {
+        var userInstance = requestUtils.toUserInstance();
+        var ticketIdentifier = requestUtils.ticketIdentifier();
         return attempt(() -> ticketService.fetchTicketForElevatedUser(userInstance, ticketIdentifier))
-                   .orElseThrow(fail -> new ForbiddenException());
+                   .orElseThrow(fail -> getForbiddenException(requestUtils));
+    }
+
+    private static ForbiddenException getForbiddenException(RequestUtils requestUtils) {
+        var publicationIdentifier = requestUtils.pathParameters().get(PUBLICATION_IDENTIFIER);
+        var ticketIdentifier = requestUtils.pathParameters().get(TICKET_IDENTIFIER);
+        logger.info(UPDATE_FORBIDDEN_MESSAGE, ticketIdentifier, publicationIdentifier, requestUtils.username());
+        return new ForbiddenException();
```

Comment:
This log message makes it possible to debug users that can do update on ticket via GUI.